common/src/compile.rs

```diff
@@ -454,9 +454,9 @@ pub fn etherscan_project(metadata: &Metadata, target_path: impl AsRef<Path>) ->
     //   ContractName/
     //     [source code]
     let paths = ProjectPathsConfig::builder()
-        .sources(sources_path)
+        .sources(sources_path.clone())
         .remappings(settings.remappings.clone())
-        .build_with_root(target_path);
+        .build_with_root(sources_path);
```

Comment:
What if there are not any direct imports? Will this break?

Reply:
It won't break anything since the compiler knows exactly where to find those, as long as those files are inside the permitted folders. And `metadata.source_tree().write_to(&target_path)` takes care of that